### PR TITLE
fix: update root cmd examples

### PR DESCRIPTION
**Pull request by sebastiendan**
# Description

This PR updates the examples listed in the root command output (the help menu).
It additionally bumps the npm package version to `1.0.2`, to prepare for new release.

## PR Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added or updated tests that comprehensively prove my change is effective or that my feature works

### CODE REVIEW

File: src/commands/root.command.ts [hunk 1]
```diff
@@ -14,26 +14,46 @@ const example = chalk.green
 const helptext = `
 Example Usage
 
-  $ ${example('topos-zkevm-demo init')}
-    Initialize the Topos-zkEVM-Demo. This command will output the status of the topos zkevm demo creation to the terminal as it runs, and will log a more detailed status to a log file.
+  $ ${example('topos-zkevm-demo install')}
+    Install Topos-zkEVM-Demo. This command will output the status of the topos zkevm demo creation to the terminal as it runs, and will log a more detailed status to a log file.
```
Comment (Nashtare) [inline src/commands/root.command.ts:18]: nit: do we want to specify the path of the log file?

File: src/commands/root.command.ts [hunk 2]
```diff
@@ -14,26 +14,46 @@ const example = chalk.green
...
 
   $ ${example('topos-zkevm-demo execute')}
     Execute the Topos-zkEVM-Demo demo script to deploy contracts and execute transactions.
 
-  $ ${example('topos-zkevm-demo get-receipt')}
-    Get a transaction receipt from a transaction hash.
+  $ ${example(
+    'topos-zkevm-demo generate merkle-proof 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5'
+  )}
+    Generate a receipt merkle proof for the 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5 transaction.
```
Comment (Nashtare) [inline src/commands/root.command.ts:29]: nit: 
```suggestion
    Generate a receipt merkle proof for the 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5 transaction hash.
```

additionally, I think it is more readable if we swap the value and the name? (same for below) but maybe that's just me

File: src/commands/root.command.ts [hunk 3]
```diff
@@ -14,26 +14,46 @@ const example = chalk.green
...
+
+  $ ${example('topos-zkevm-demo generate zk-proof 2')}
+    Generate a zk-proof for block 2.
+
+  $ ${example(
+    'topos-zkevm-demo util get-receipt-trie-root 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5'
+  )}
+    Get the receipt trie root of the block of the 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5 transaction.
```
Comment (Nashtare) [inline src/commands/root.command.ts:37]:
```suggestion
    Get the receipt trie root of the block containing the transaction with hash 0x064eaad7fb35c25410aec7f55603d9bb95436891cbda68a4be9a6fc3b19ca0a5.
```

**Review by Nashtare — APPROVED** [review]
LGTM! I just have some minor nits/questions regarding help sentences.

File: src/commands/root.command.ts (hunk 1 above)
Comment (sebastiendan) [reply src/commands/root.command.ts:18]: The path is detailed in the output of the command

File: src/commands/root.command.ts (hunk 2 above)
Comment (Nashtare) [reply src/commands/root.command.ts]: (same for below) also was applicable to the `verify merkle-proof` piece, but I guess that's ok :)

**Pull request merged**